deeplearning4j/deeplearning4j-nn/src/main/java/org/deeplearning4j/nn/layers/normalization/BatchNormalization.java

```diff
@@ -72,11 +73,33 @@ void initializeHelper() {
         String backend = Nd4j.getExecutioner().getEnvironmentInformation().getProperty("backend");
 
         if ("CUDA".equalsIgnoreCase(backend)) {
-            helper = DL4JClassLoading.createNewInstance(
-                    "org.deeplearning4j.cuda.normalization.CudnnBatchNormalizationHelper",
-                    BatchNormalizationHelper.class,
-                    dataType);
-            log.debug("CudnnBatchNormalizationHelper successfully initialized");
+            if(DL4JClassLoading.loadClassByName(BATCH_NORM_CUDNN_HELPER_CLASS_NAME) != null) {
+                helper = DL4JClassLoading.createNewInstance(
+                        BATCH_NORM_CUDNN_HELPER_CLASS_NAME,
+                        BatchNormalizationHelper.class,
+                        dataType);
+                log.debug("CudnnBatchNormalizationHelper successfully initialized");
+
+            } else {
+                log.warn("Cudnn class not found using current class loader. Trying current classloader.");
```

Comment:
And a fourth time... Had I known that this thing repeats so often, I wouldn't have flagged every single appearance of it.